file_util/tests/file_utils_test.cc

```diff
@@ -0,0 +1,80 @@
+#include <catch.hpp>
+
+#include <leatherman/file_util/file.hpp>
+
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
+namespace leatherman { namespace file_util {
+
+// TODO(ale): consider making file_utils.cpp and string_utils.cpp
+// consistent across cthun-agent and pegasus before writing more tests
+
+TEST_CASE("file_util::tildeExpand", "[utils]") {
+#ifdef _WIN32
+    _putenv("USERPROFILE=/testhome");
```

Comment:
Found the documentation on setenv, but I'm still unclear on how to use it here...